### ISSUE TITLE
Should this package be on CRAN?

### ISSUE THREAD

**Issue by seasmith** [seasmith/AlignAssign#9]
I never intended this to be on CRAN but parts of the code base live in the remedy package.  Extra features have been added to AlignAssign since remedy made its way to CRAN.

**Comment by emmansh** [comment]
Yes, please! It will be outstanding for those of us who work under firewall limitations. At least in my case, CRAN/[MRAN](https://mran.microsoft.com/packages) is considered legit but github isn't :(

**Comment by seasmith** [comment]
* [R-hub CRAN check](https://r-hub.github.io/rhub/reference/check_for_cran.html)
* [CRAN checklist](https://cran.r-project.org/web/packages/submission_checklist.html)
